docs/using_deis/using-docker-images.rst

```diff
@@ -30,14 +30,10 @@ Docker Image Requirements
 ^^^^^^^^^^^^^^^^^^^^^^^^^
 In order to deploy Docker images, they must conform to the following requirements:
 
- * The Docker image must EXPOSE only one port
- * The exposed port must be an HTTP service that can be connected to an HTTP router
+ * The Docker image must EXPOSE at least one port
```

Comment:
Not sure how to clarify that official multi-port images (i.e exposing two HTTP processes in a container, or a web and an SSH process once #1315 is tackled) are still unsupported. Any help on this would be appreciated :)